get-config.js

```diff
@@ -0,0 +1,21 @@
+function readConfig() {
+	try {
+		return require( './config' )();
```

Comment:
@dmsnell to be backwards compatible we can use `typeof` and execute if it's a function but still update the README to show exporting of an object literal.